js/capture.js

```diff
@@ -10,10 +10,25 @@ const capture = {
   addListeners() {
     // listen for each HTTP response
     browser.webRequest.onResponseStarted.addListener(
-      this.sendThirdParty,
+      (response) => this.sendThirdParty(response),
```

Comment:
Please add a follow up bug to make this use braces. Having the implied return may in future cause issues as this method returns a promise...blocking requests for example would delay.